recipes/frogs/meta.yaml

```diff
@@ -61,8 +61,8 @@ about:
   home: https://github.com/geraldinepascal/FROGS
   license: 'GNU GPL v3'
   summary: 'FROGS is a workflow designed to metabarcoding sequence analysis'
-  description: 'FROGS produces an OTUs count matrix from high depth sequencing amplicon data. This is the official release 3.2.3 of FROGS.
-  To fully install FROGS dependencies, please refer to the frogs-conda-requirements.txt available at https://github.com/geraldinepascal/FROGS'
+  description: 'FROGS produces an OTUs count matrix from high depth sequencing amplicon data. This is the official release 4.0.1 of FROGS.
+  To fully install FROGS dependencies, please refer to the frogs-conda-requirements.txt and frogsfunc-conda-requirements.txt available at https://github.com/geraldinepascal/FROGS'
```

Comment:
@vindarbot why is this needed? What mean full dependencies, is this package not working?

Reply:
Dear Björn,

FROGS is a pipeline that request other dependancies such as swarm, vsearch, blast, some R packages.

We share yaml file to install all this dependencies with fixed versions that we tested.

Since FROGS 4.0.0 we added functionnal tools, which rely on Picrust suite that unfortunately has conflicts on some R packages version ( DESeq2 in particular that we fixed with you : https://github.com/geraldinepascal/FROGS-wrappers/pull/13).
So we just create an other yaml file for these 4 new tools (and update the documentation here that I forget to do when publshing the frogs 4.0.0 documentation).

Reply:
@bgruening, does it make sense to you?